os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c

```diff
@@ -851,13 +894,54 @@ _timerHandle _tizenrt_timerCreate(const signed char *pcTimerName, osdepTickType
 	timer->timevalue = xTimerPeriodInTicks;
 	timer->data = pvTimerID;
 	timer->function = pxCallbackFunction;
+
+	if (!_tizenrt_timer_table_init) {
+		INIT_LIST_HEAD(&_tizenrt_timer_table);
+		_tizenrt_timer_table_init = 1;
+	}
+
+	struct _tizenrt_timer_entry *timer_entry;
+	timer_entry = _tizenrt_zmalloc(sizeof(struct _tizenrt_timer_entry));
+	if (timer_entry == NULL) {
+		kmm_free(timer->work_hdl);
+		kmm_free(timer);
+		return NULL;
+	}
+	memset(timer_entry, 0, sizeof(struct _tizenrt_timer_entry));
+	timer_entry->timer = timer;
```

Comment:
`timer_entry` is set to 0 from line 910.
But in here, you used its pointer, `timer_entry->timer`. It looks `NULL->xxx`.
I think it can make assert.
Could you check this?

Reply:
There is no assert, Line 910 is just initialize the whole structure to 0.
In 911, timer_entry->timer is been assign to timer, which is a pointer.

struct _tizenrt_timer_entry {
	struct list_head list;
	struct timer_list_priv *timer;
};